setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import find_packages, setup
+
+from BeamEnergy.version import __version__, licence
+from BeamEnergy import __doc__, __author__, __author_email__
+
+setup(
+    name="beam_energy",
+    author=__author__,
+    author_email=__author_email__,
+    version=__version__,
+    license=licence,
+    description="A simple facade device for calculating beam's energy",
+    long_description=__doc__,
+    url="https://github.com/synchrotron-solaris/dev-solaris-beamenergy.git",
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=["setuptools"],
+    entry_points={
+        "console_scripts": ["BeamEnergy = "
+                            "BeamEnergy.BeamEnergy:run"]}
```

Comment:
As it's written in the coding policies (a note on using POGO generated things), I'd like to ask you to rename the modules inside your package so that there are no imports like
```python
from A.A import A
```
I'd suggest the following:
* name the top-level directory `beam_energy_ds` and change the `name` in `setup` accordingly
* name the Python file with the `BeamEnergy` class `beam_energy` (so using `snake_case`)

That way we'll have imports like:
```python
from beam_energy_ds.beam_energy import BeamEnergy
```
And that looks a lot cleanier.